### PR TITLE
feat: Added so that island update include previous islands

### PR DESCRIPTION
**Pull request by pablitar**

### CODE REVIEW

File: test/lib.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ export function setMultiplePeersAround(
   position: Position3D,
   qty: number,
   idGenerator: IdGenerator = sequentialIdGenerator("P"),
-  offset: Position3D = [20, 0, 20]
+  offset: Position3D = [10, 0, 10]
```
Comment (nchamo) [inline test/lib.ts:57]: Curious, why was this change necessary?

**Review by nchamo — APPROVED** [review]
Great work!

File: test/lib.ts (hunk 1 above)
Comment (pablitar) [reply test/lib.ts:57]: The test was a little flaky, because it has random positions. Reducing the offset seems to make it more reliable.

Eventually we may want to use a known seed to make the tests deterministic.
